astropy/io/fits/fitstime.py

```diff
@@ -241,21 +329,37 @@ def _convert_time_column(col, column_info):
         The time coordinate column to be converted to Time.
     column_info : dict
         Column-specific time reference frame override information.
+    global_info : dict
+        Global time reference frame information.
     """
 
     # The code might fail while attempting to read FITS files not written by astropy.
 
-    # Read in time coordinate column as Time
-    if col.shape[-1] == 2 and col.ndim > 1:
-        col = Time(col[..., 0], col[..., 1], format='jd',
-                   scale=column_info['scale'],
-                   location=column_info['location'])
-    else:
-        warnings.warn(
-            'Time column {} is not in the astropy required (jd1, jd2) format. '
-            'Hence, it will not be read as astropy Time'.format(col.info.name),
-            AstropyUserWarning)
-    return col
+    try:
+        # ISO-8601 is the only string representation of time in FITS
+        if col.info.dtype.kind == 'S':
```

Comment:
Quite a lot of things in this code are specified in the standard but have no valid examples which I can refer to. So do let me know if some interpretation has gone haywire. I think it would get recognised by the scale as you mentioned, since ISO-8601 does not imply a particular time scale.